src/mainboard/protectli/vault_adl_n/variants/vp2430/gpio.c

```diff
@@ -0,0 +1,1415 @@
+	/* ------- GPIO Group GPP_F ------- */
+
+	/* GPP_F0 - CNV_BRI_DT */
+	/* DW0: 0x44000700, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_F0, NONE, DEEP, NF1), */
+	_PAD_CFG_STRUCT(GPP_F0, PAD_FUNC(NF1) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE), 0),
+
+	/* GPP_F1 - CNV_BRI_RSP */
+	/* DW0: 0x44000702, DW1: 0x00003000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) | (1 << 1) - IGNORED */
+	/* PAD_CFG_NF(GPP_F1, UP_20K, DEEP, NF1), */
+	_PAD_CFG_STRUCT(GPP_F1, PAD_FUNC(NF1) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) | (1 << 1), PAD_PULL(UP_20K)),
+
+	/* GPP_F2 - CNV_RGI_DT */
+	/* DW0: 0x44000700, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_F2, NONE, DEEP, NF1), */
+	_PAD_CFG_STRUCT(GPP_F2, PAD_FUNC(NF1) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE), 0),
+
+	/* GPP_F3 - CNV_RGI_RSP */
+	/* DW0: 0x44000702, DW1: 0x00003000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) | (1 << 1) - IGNORED */
+	/* PAD_CFG_NF(GPP_F3, UP_20K, DEEP, NF1), */
+	_PAD_CFG_STRUCT(GPP_F3, PAD_FUNC(NF1) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) | (1 << 1), PAD_PULL(UP_20K)),
+
+	/* GPP_F4 - CNV_RF_RESET# */
+	/* DW0: 0x44000700, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_F4, NONE, DEEP, NF1), */
+	_PAD_CFG_STRUCT(GPP_F4, PAD_FUNC(NF1) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE), 0),
+
+	/* GPP_F5 - MODEM_CLKREQ */
+	/* DW0: 0x44000b00, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_F5, NONE, DEEP, NF2), */
+	_PAD_CFG_STRUCT(GPP_F5, PAD_FUNC(NF2) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE), 0),
+
+	/* GPP_F6 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_F6, NONE),
+
+	/* GPP_F7 - GPIO */
+	/* DW0: 0x44000200, DW1: 0x00000000 */
+	PAD_CFG_GPO(GPP_F7, 0, DEEP),
+
+	/* GPP_F8 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_F8, NONE),
+
+	/* GPP_F9 - BOOTMPC */
+	/* DW0: 0x44000700, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_F9, NONE, DEEP, NF1), */
+	_PAD_CFG_STRUCT(GPP_F9, PAD_FUNC(NF1) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE), 0),
+
+	/* GPP_F10 - GPIO */
+	/* DW0: 0x44000200, DW1: 0x00000000 */
+	PAD_CFG_GPO(GPP_F10, 0, DEEP),
+
+	/* GPP_F11 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_F11, NONE),
+
+	/* GPP_F12 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_F12, NONE),
+
+	/* GPP_F13 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_F13, NONE),
+
+	/* GPP_F14 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_F14, NONE),
+
+	/* GPP_F15 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_F15, NONE),
+
+	/* GPP_F15 - GPIO (5G Power_On_Off)*/
+	PAD_CFG_GPO(GPP_F15, 0, DEEP),
```

Comment:
Copy pasta error? This pin is not connected anywhere on the schematics

Reply:
[Removed](https://github.com/Dasharo/coreboot/pull/576/commits/68fe32358db1c4826321169730cb09ec89ec4a40#diff-2f9e6fbf9293a4dbeecc2314eb0b449c4eb64691bfc1c58981fc506a602441bcL1169)